otterdog/eclipse-sw360.jsonnet

```diff
@@ -10,11 +10,19 @@ local customRuleset(name) =
     ],
     required_pull_request+: {
       required_approving_review_count: 1,
-      requires_last_push_approval: true,
-      requires_review_thread_resolution: true,
       dismisses_stale_reviews: true,
+      requires_code_owner_reviews: true,
+      requires_last_push_approval: true,
+      requires_review_thread_resolution: true
     },
+    required_approving_review_count: 1,
     requires_linear_history: true,
+    requires_conversation_resolution: true,
```

Comment:
the settings that you added here are named slightly differently for rulesets and are already there ins the required_pull_request object above.

Reply:
done